.github/workflows/flake8.yml

```diff
@@ -0,0 +1,24 @@
+name: Flake8 Check
+
+on:
+  pull_request:
+    branches:
+      - master
```

Comment:
this branch doesn't exists, actually I think this PR should run the action

Reply:
Done